modules/tooltip/react/stories/Tooltip.stories.mdx

```diff
@@ -0,0 +1,130 @@
+import {Meta} from '@storybook/addon-docs/blocks';
+
+import {Tooltip} from '@workday/canvas-kit-react-tooltip';
+import {Specifications} from '@workday/canvas-kit-specifications';
+
+import {Default} from './examples/Default';
+import {CustomContent} from './examples/CustomContent';
+import {DescribeType} from './examples/DescribeType';
+import {Placements} from './examples/Placements';
+import {Ellipsis} from './examples/Ellipsis';
+import {LineClamp} from './examples/LineClamp';
+import {UseTooltip} from './examples/UseTooltip';
+
+<Meta title="Components/Popups/Tooltip/React" component={Tooltip} />
+
+# Canvas Kit React Tooltips
+
+A Tooltip component that renders information/text when the user hovers over an element. A tooltip is
+used to label or describe an element. By default, a tooltip will label an element. This is useful
+for IconButtons. A tooltip can also be used to describe additional information about an element
+
+[> Workday Design Reference](https://design.workday.com/components/popups/tooltips)
+
+## Installation
+
+```sh
+yarn add @workday/canvas-kit-react
+```
+
+or
+
+```sh
+yarn add @workday/canvas-kit-react-tooltip
+```
+
+## Usage
+
+This component follows the
+[W3 Tooltip specification](https://www.w3.org/TR/wai-aria-practices/#tooltip). Tooltips are used to
+label icon buttons and provide additional context to elements.
+
+### Basic Example
+
```

Comment:
Could we add a kind of decision flow chart for when to use which type of tooltip here? I've written up this, but happy to take it to the content team if we feel it needs some sprucing up.

When to use tooltips:
If you’re trying to display additional information for users to better understand the purpose, context, or interaction.

When not to use tooltips:
When the visual text will be the exact same as what is visually displayed to the user without the tooltip being visible

Tooltip Usage Process
1. Does this widget need additional context or information
    1. If no, don’t use a tooltip
2. Is the tooltip text different from visual text displayed to users
    1. If there is no text, use aria-label (icon example)
    2. If yes, use aria-describedby
    3. If no, don’t use a tooltip
3. Is the tooltip text useful to screen reader users?
    1. If no, use muted tooltip

Reply:
Perfect! I'll also added the `muted` type and some tests and examples.

Reply:
Is there a spot in that decision tree for a label to provide additional context that a visual user wouldn't need? For example, if a card titled `Dependents` contains a button to add a new dependent, would the button text be `Add New` while the `aria-label="Add New Dependent"`? Or would that be difficult to handle for translations?

It is a balance from the visual being too noisy and buttons not having enough context in isolation (i.e. VO's web rotor lists).

Reply:
Nicholas that is a good call out. Maybe we chat about that one? I was thinking in your example I think we'd just want a muted tooltip. Since the aria-label for the button could be "add new dependent" and the tooltip probably isn't needed at that point. I guess if it was an icon button that had a "+" and then aria-label was "add new dependent" we'd want a muted tooltip since the visual tooltip wouldn't add any additional information. If you think a quick 5 minute chat would be useful let me know.